### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cdaniluk**

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: check
```
Comment (sblack4) [inline .github/workflows/check.yml]:
```suggestion
name: format_and_lint
```
the name should be descriptive. I've started using a `pre-commit` workflow to shame all who don't conform

```
name: pre-commit-check
on:
  push:
    branches:
      - master
  pull_request:

jobs:
  build:
    runs-on: macOS-latest
    steps:
    - uses: actions/checkout@v1

    - name: Install prerequisites
      run: |
        brew install tfenv tflint terraform-docs pre-commit
        pre-commit install
        tfenv install
    - name: pre-commit run all
      run: |
        pre-commit run -a
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# terraform-aws-fortigate-vpn
+
```
Comment (sblack4) [inline README.md]:
```suggestion
[![GH Action Status](https://github.com/rhythmictech/terraform-aws-fortigate-vpn/workflows/check/badge.svg)](https://github.com/rhythmictech/terraform-aws-fortigate-vpn/actions)
```
add button for swag

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      tunnel1_address                 = aws_vpn_connection.this.tunnel1_address
+      tunnel1_inside_address_amazon   = aws_vpn_connection.this.tunnel1_vgw_inside_address
+      tunnel1_inside_address_customer = aws_vpn_connection.this.tunnel1_cgw_inside_address
+      tunnel2_address                 = aws_vpn_connection.this.tunnel2_address
+      tunnel2_inside_address_amazon   = aws_vpn_connection.this.tunnel2_vgw_inside_address
+      tunnel2_inside_address_customer = aws_vpn_connection.this.tunnel2_cgw_inside_address
+      wan_interface                   = var.wan_interface
+    }
+  )
+  filename = "${var.account_name}-${var.customer_name}-fortigate_config.txt"
```
Comment (sblack4) [inline main.tf:84]: ❤️

File: outputs.tf [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+output "vpn_connection_tunnel2_cgw_inside_address" {
+  description = "Tunnel 2 Customer Inside IP Address"
+  value       = aws_vpn_connection.this.tunnel2_cgw_inside_address
+}
+
+output "vpn_connection_tunnel2_preshared_key" {
+  description = "Tunnel 2 Preshared Key"
+  value       = aws_vpn_connection.this.tunnel2_preshared_key
```
Comment (sblack4) [inline outputs.tf:57]: are the keys sensitive?

**Review by sblack4 — APPROVED** [review]
:shipit: , small suggestions

File: outputs.tf (hunk 4 above)
Comment (cdaniluk) [reply outputs.tf:57]: yes

Comment (cdaniluk) [reply outputs.tf:57]: replaced with secret arn

**Pull request merged**